components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -884,6 +885,15 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
             });
             throw err;
         }
+        const filtered = this.state[deploymentNameKey].match(nameformat);
+        if (!(filtered && this.state[deploymentNameKey] === filtered[0])) {
+            this.setState({
+                dialogAsCode: false,
+                dialogBody: 'Deployment name need to match rgular expression: [a-z]([-a-z0-9]*[a-z0-9])?',
```

Comment:
s/rgular/regular/

more importantly, should we make this more human readble. ie., name needs to be 25 chars or less of lowercase alphanumeric characters and hyphens.

Reply:
updated